whisper/transcribe.py

```diff
@@ -205,7 +208,7 @@ def decode_with_fallback(segment: torch.Tensor) -> DecodingResult:
                 compression_ratio_threshold is not None
                 and decode_result.compression_ratio > compression_ratio_threshold
             ):
-                needs_fallback = True  # too repetitive
+                needs_fallback = True  # too repetitive <-- We can inprove it...
```

Comment:
"inprove" should be "improve" (although to be honest I'm not sure what this entire comment is trying to convey)

Reply:
Forgot this comment fixed in the latest commit